linters/.markdownlint.json

```diff
@@ -0,0 +1,33 @@
+{
+  "comment": "Enable all rules, and then disable selective ones https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+  "default": true,
+
+  "line-length": false,
+  "ul-start-left": false,
+  "no-multiple-blanks": false,
+  "no-emphasis-as-header": false,
```

Comment:
This rule triggers whenever there is as paragraph that consists only of emphasized text.

Some examples are the document subhead

```
*A mostly reasonable approach to JavaScript*
```

and the sections under `##Resources` like

```
**Read This**

  - [Standard ECMA-262](http://www.ecma-international.org/ecma-262/6.0/index.html)
```

(which seem like they probably should either be list items or headings).

Here's the complete list.

```
README.md: 3: MD036 Emphasis used instead of a header
README.md: 3158: MD036 Emphasis used instead of a header
README.md: 3165: MD036 Emphasis used instead of a header
README.md: 3169: MD036 Emphasis used instead of a header
README.md: 3177: MD036 Emphasis used instead of a header
README.md: 3183: MD036 Emphasis used instead of a header
README.md: 3190: MD036 Emphasis used instead of a header
README.md: 3198: MD036 Emphasis used instead of a header
README.md: 3217: MD036 Emphasis used instead of a header
README.md: 3230: MD036 Emphasis used instead of a header
react/README.md: 3: MD036 Emphasis used instead of a header
```